### PR TITLE
HPCC-15301 Removing ./ from despray

### PR DESCRIPTION
**Pull request by miguelvazq**
Fixing a commit https://github.com/hpcc-systems/HPCC-Platform/pull/8407/files that was overwriten with another commit.

Signed-off by: Miguel Vazquez miguel.vazquez@lexisnexis.com

### CODE REVIEW

**Comment by miguelvazq** [comment]
@GordonSmith please review.

**Comment by GordonSmith** [comment]
Looks fine

File: esp/src/eclwatch/TargetSelectClass.js [hunk 1]
```diff
@@ -239,7 +240,7 @@ define([
         loadDropZoneFolders: function () {
             var context = this;
             this.getDropZoneFolder = function () {
-                var baseFolder = this._dropZoneTarget.machine.Directory + (this.endsWith(this._dropZoneTarget.machine.Directory, "/") ? "" : "/");
+                var baseFolder = this._dropZoneTarget.machine.Directory + (this.endsWith(this._dropZoneTarget.machine.Directory, "/") ? "" : "");
```
Comment (AttilaVamos) [inline esp/src/eclwatch/TargetSelectClass.js]: Maybe I am wrong, but does the second argument of this addition have any sense?

Comment (GordonSmith) [inline esp/src/eclwatch/TargetSelectClass.js]: Yup, I agree

Comment (miguelvazq) [inline esp/src/eclwatch/TargetSelectClass.js]: I'll take a look

**Comment by miguelvazq** [comment]
@GordonSmith I removed the endswith call as it is not needed any longer. I also added a `required:false` to the LFDetailsWidget as there is a despray that can occur there as well.

**Comment by GordonSmith** [comment]
Looks fine.

**Pull request merged**